### DISCUSSION TITLE
Quick Switcher (like Discord CTRL-K)

### DISCUSSION THREAD

**Discussion by Coldblackice** [Chatterino/chatterino2#3594] [Ideas] [answered]
A phenomenal feature would be a Discord-like "Quick Switcher": a hotkey pulls up a floating overlay or focuses a textbox, input into which jumps you to a particular channel/tab you have open, and if not already open, offers the option to open that channel in a new tab.

With many channels open, it can get difficult to know if you already have a particular channel open. Please correct me if I'm overlooking an ability that addresses this.

**Comment by Mm2PL** [comment, marked as answer]
This already exists https://github.com/Chatterino/chatterino2/pull/1588

Last changed in: https://github.com/Chatterino/chatterino2/pull/2340 and https://github.com/Chatterino/chatterino2/pull/2561

**Reply by Coldblackice** [reply]
Yikes, my bad! Near verbatim, too. Thanks for the heads-up.

**Comment by leon-richardt** [comment]
Ah yes, Chatterino's famed discoverability strikes once again

**Comment by Mm2PL** [comment]
Yeah need to so something about that

8 mar 2022 12:29:39 Leon Richardt ***@***.***>:
